src/lib/constants.ts

```diff
@@ -9,7 +9,13 @@ export const TRANSLATED_IMAGES_DIR = "/content/translations"
 // i18n
 export const DEFAULT_LOCALE = "en"
 // Sorted list of supported locales codes, defined in `i18n.config.json`
-export const LOCALES_CODES = i18nConfig.map((lang) => lang.code).sort()
+const BUILD_LOCALES = (process.env.BUILD_LOCALES || "")
+  .split(",")
+  .filter((item) => item.length > 1)
+export const LOCALES_CODES =
+  BUILD_LOCALES.length > 0
+    ? BUILD_LOCALES.sort()
+    : i18nConfig.map((lang) => lang.code).sort()
```

Comment:
Would align this with the exact same code we end up using in the `next.config.js` file..